webapp/pages/post/_id/_slug/index.vue

```diff
@@ -90,7 +90,17 @@
           @toggleNewCommentForm="toggleNewCommentForm"
         />
         <ds-space margin-bottom="large" />
-        <hc-comment-form v-if="showNewCommentForm" :post="post" @createComment="createComment" />
+        <comment-form
+          v-if="showNewCommentForm && !post.author.isBlocked"
```

Comment:
I'm just looking at the frontend right now, so not sure where this is coming from – but this reads like "when I have blocked the post author" when it should be "when the post author has blocked me". Or am I wrong?

Reply:
it is reciprocal @alina-beck ... we could think about renaming it to make it clear... 
maybe `blockedExists` or `hasBlocked`
`post.authord.hasBlocked` works well, but `user.hasBlocked`, maybe not so much??

Reply:
I just renamed it to `blocked`, which could stand for `isOrWasBlocked` ... didn't really love any of the names I could come up with to describe it..